chat: add API reference landing page (#2908)
ably/docs: src/pages/docs/chat/api/index.mdx

---
title: Ably Chat API Reference

> meta_description: "API Reference section of the Ably Chat developer documentation."

Author: I'm going to update the pubsub one too as I copied this wholesale :D

> meta_keywords: "Ably, Ably realtime, API Reference, Chat SDK, Ably Chat, REST SDK, REST API, SSE API, Control API"

Reviewer: Some of these aren't relevant to Chat.

---

This section of the documentation contains the API references for Ably Chat.

## Client library SDKs <a id="sdks" />

The following API references are available. They are automatically generated from source code:

* [JavaScript](https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/modules/chat-js.html)
* [React Hooks](https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/modules/chat-react.html)
* [Kotlin (JVM & Android)](https://sdk.ably.com/builds/ably/ably-chat-kotlin/main/dokka/)
* [Swift](https://sdk.ably.com/builds/ably/ably-chat-swift/main/AblyChat/documentation/ablychat/)

## REST API <a id="rest-api" />

> A raw [REST API](/docs/api/chat-rest) is also provided - you don't need to install any SDK to use this. This is used where an Ably Chat client library SDK is not available for your chosen platform, or you do not wish to use a client library SDK due to resource constraints.

Reviewer: ```suggestion
A raw [REST API](/docs/api/chat-rest) is also provided - you don't need to install any SDK to use this. This is used where an Ably Chat SDK is not available for your chosen platform, or you don't wish to use an SDK due to resource constraints.
```

## Further information <a id="further-information" />

> In addition to the API references listed previously, our developer documentation also provides information on how these interfaces are used, and this covers key concepts such as connections, chat rooms, messages and typing indicators. You can find that information on the following pages:

Reviewer: ```suggestion
In addition to the API references listed, Ably's documentation provides information on how these interfaces are used. This includes key concepts such as connections, chat rooms, messages and typing indicators. You can find that information on the following pages:
```
